packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -402,11 +456,29 @@ class InputDecorator extends StatelessWidget {
       );
     }
 
+    Widget inputChild;
+    if (!hasInlineLabel && (!isEmpty || hintText == null) &&
+        (decoration?.prefixText != null || decoration?.suffixText != null)) {
+      final List<Widget> rowContents = <Widget>[];
+      if (decoration.prefixText != null) {
+        rowContents.add(new Text(decoration.prefixText,
```

Comment:
dartfmt doesn't implement Flutter style so we can't use it. We are looking into other options but don't currently have one.
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo describes Flutter style.